rules/test-rule/rule.md

```diff
@@ -0,0 +1,54 @@
+---
+type: rule
+title: Induction task - Do you know this is a test rule?
+guid: df9552b7-1eff-4acc-9ae5-8e3db2d3cc82
+uri: test-rule
+created: 2023-01-23T15:23:10Z
+authors:
+  - title: Hugo Pernet
+    url: https://ssw.com.au/people/hugo-pernet
+---
+This is the introduction to my awesome rule.
+<!--endintro-->
+# Lists
+* I
+* Really
+* Like
+* Lists
+
+1. But
+2. I
+3. Prefer
+4. Them
+5. Ordered
+
+#Link, Image and Video
+
+Check out this [link](https://www.nintendo.fr/Jeux/Super-Nintendo/The-Legend-of-Zelda-A-Link-to-the-Past-841179.html "got it?"), it's clearly the best.
+
+::: img-small  
+![Cool image](test-img.jpg)
+:::
+
+Cool User Group teaser video:
+`youtube: https://www.youtube.com/embed/0akg1RqVQFo`
+
```

Comment:
Can you include a caption as per https://ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-use-the-titles-for-video-captions ?

Reply:
Done - added a caption